uploadNeuroDB/imaging_upload_file_cronjob.pl

```diff
@@ -72,7 +73,9 @@
 ################################################################
 my $dbh = &NeuroDB::DBI::connect_to_db(@Settings::db);
 my @row=();
-my $query = "SELECT UploadID, UploadLocation FROM mri_upload WHERE Processed=0 AND (TarchiveID IS NULL AND number_of_mincInserted IS NULL)";
+my $query = "SELECT UploadID, UploadLocation FROM mri_upload 
+		WHERE Inserting <> 1 AND InsertionComplete <> 1 
+		AND (TarchiveID IS NULL AND number_of_mincInserted IS NULL)";
 print "\n" . $query . "\n";
```

Comment:
That's a good point but I believe we should be consistent none of the codes written for imaging-uploader uses '<QUERY'. And I personally prefer concatenation of the strings (i.e. "QUERY" . "QUERY")